src/main/controllers/ContactAcasController.ts

```diff
@@ -0,0 +1,10 @@
+import { Request, Response } from 'express';
+
+export default class ContactAcasController {
+  public get(req: Request, res: Response): void {
```

Comment:
Could you please use AppRequest rather than Request. It extends Request and I think it should be the default going forward

Reply:
updated